Bluejay/Bluejay/SynchronizedPeripheral.swift

```diff
@@ -90,23 +90,35 @@ public class SynchronizedPeripheral {
     }
     
     /// Listen for changes on a specified characterstic synchronously.
-    public func listen<R: Receivable>(to characteristicIdentifier: CharacteristicIdentifier, completion: @escaping (R) -> ListenAction) throws {
+    public func listen<R: Receivable>(
+        to characteristicIdentifier: CharacteristicIdentifier,
+        timeoutInSeconds: Int = 0,
```

Comment:
I fell like this should be using the same timeout enum that the connect timeout uses. Is there a reason not to?

Reply:
It should, I forgot about that enum.